cloudwatchlogs-with-dlq/DLQProcessor.js

```diff
@@ -16,8 +16,17 @@ exports.consumeMessages = function (env, context, callback) {
             console.log("Messages Received", messages.length);
             for (var i = 0; i < messages.length; i++) {
                 (function(idx) {
-                    var logdata = JSON.parse(messages[idx].Body).awslogs.data;
+                    var payload = JSON.parse(messages[idx].Body);
                     var receiptHandle = messages[idx].ReceiptHandle;
+                    if (!(payload.awslogs && payload.awslogs.data)) {
```

Comment:
Should test if this fails fast (ie in case awslogs doesn't exist, does it also check awslogs.data).

Reply:
Tested myself, it does fail fast